data.csv

```diff
@@ -42,4 +42,5 @@ Figma,2016-09-27,Penpot,2021-02-02,1589 days
 Canny.io,2017-03-01,Astuto,2019-08-19,901 days
 Roam Research,2019-10-14,Foam,2020-06-14,244 days
 Clubhouse,2020-03-01,Dogehouse,2021-02-07,343 days
+SAS,1970-01-01,R,1993-08-01,8613 days
```

Comment:
```suggestion
S Language,1976-01-01,R Language,1993-08-01,6422 days
```

It seems that R was an open-source implementation of [S](https://en.wikipedia.org/wiki/S_(programming_language)) which first appeared in 1976.

Reply:
That’s a good point. I was thinking that R is sort of a drop in replacement for SAS but R is literally an OSS version of S.